### PR TITLE
Huge ships

### PR DESCRIPTION
**Pull request by therealppa**
I scraped some basic data for the huge ships. I would also like to add some features to the processing script to add more data, but maybe this could already be valuable as a start?

### CODE REVIEW

**Comment by therealppa** [comment]
I'll take a look at the tests and add the medatory fields

File: data/manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.0.4",
+  "version": "2.0.5",
```
Comment (guidokessels) [inline data/manifest.json:2]: No need to bump the manifest in this commit. We should bump it together with the version number in package.json when a new release is created.

Comment (therealppa) [reply data/manifest.json]: Should be fixed now.

**Comment by guidokessels** [comment]
Thanks a lot for this!

**Comment by therealppa** [comment]
> Thanks a lot for this!

You're welcome :)

**Pull request merged**